apps/mobile/src/util/navigation.js

```diff
@@ -49,12 +45,10 @@ export function baseUrl ({
     return viewUrl(view, { context, customViewId, defaultUrl, groupSlug })
   } else if (groupSlug) {
     return groupUrl(groupSlug)
-  } else if (context === ALL_GROUPS_CONTEXT_SLUG) {
```

Comment:
Bookmarking this

Reply:
ok I suspect this will get us into trouble.

Some widgets will be passed in with an `all` context.

So their paths won't render correctly and they won't be caught by the linking table.

We can change that in the staticContextGroups, by editing the static widgets...

So then their paths are just substituting `my` in the place of `all` and the path still got us to the Stream screen, then the query will fail, as far as I can tell from this changeset, because there will be nothing indicating to the stream query that it will match with the backend's expectations for the query

If we change this backend line

`onlyMyGroups: context === 'all',` to something like 
`onlyMyGroups: context === 'all' || context === 'my', then we are probably safe.

Reply:
I agree with your suspicions, but not yet with the solution. Will
loop back more.